wp-content/themes/twentyseventeen-wp20/style.css

```diff
@@ -219,6 +221,23 @@ input[type="submit"]:focus {
  * Layout
  */
 
+#page {
+	min-height: 100vh;
```

Comment:
A downside of this is that part of the footer is always below the fold:

![footer always below fold](https://user-images.githubusercontent.com/484068/231825661-abe8d346-537c-4967-b026-1bb13e7fb38f.gif)

Something like `calc( 100vh - 35px )` may be enough to solve that, if we can assume that the footer always has a fixed height.

If not, then IMO this is good enough for launch, and we can iterate after. I'm actually not sure that we need this at all, since I don't think we'll have any pages on production that are short enough to expose it. Now that the Live page has dynamic content again, the Swag page at `1028px` seems like the shortest any page will ever be. 

A post could theoretically be shorter, but that seems unlikely. [Selfie Props](https://wp20.wordpress.net/download-selfie-props-for-wp20-meetup-celebrations/) is the shortest one I see, and it's still long enough.

Reply:
> Something like calc( 100vh - 35px ) may be enough to solve that, if we can assume that the footer always has a fixed height.

Nice catch. It's actually the admin bar which is causing this, and I don't have it enabled for my user so didn't see it. I think this will fix it:

```
.admin-bar #page {
    min-height: calc(100vh - 32px);
}
```

Reply:
Fixed this footer issue, and the horizontal overflow issues.

Reply:
Ah, that sounds good 👍🏻 

Using `--wp-admin--admin-bar--height` would be even better, since the height changes a bit based on the breakpoint.

Reply:
Ah I wondered if there was a var, I'll use that, thanks

Reply:
✅ Updated